sky/services/dynamic/BUILD.gn

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+source_set("embedder") {
+  sources = [
+    "dynamic_service.c",
+    "dynamic_service.h",
+    "dynamic_service_definition.cc",
+    "dynamic_service_definition.h",
+    "dynamic_service_embedder.c",
+    "dynamic_service_embedder.h",
+    "dynamic_service_macros.h",
+  ]
+
+  deps = [
+    "//mojo/public/c/environment:environment",
+    "//mojo/public/c/system:system",
```

Comment:
"//mojo/public/cpp/environment:standalone" and "//mojo/public/platform/native:system" were required since they are not being added using the default rule. I have removed the rest of the redundancies in this file. Thanks.